Mathlib/Data/Real/GoldenRatio.lean

```diff
@@ -233,3 +243,26 @@ theorem Real.coe_fib_eq' :
 theorem Real.coe_fib_eq : ∀ n, (Nat.fib n : ℝ) = (φ ^ n - ψ ^ n) / Real.sqrt 5 := by
   rw [← Function.funext_iff, Real.coe_fib_eq']
 #align real.coe_fib_eq Real.coe_fib_eq
+
+/-- Relationship between the Fibonacci Sequence, Golden Ratio and its conjugate's exponents --/
+theorem fib_golden_conj_exp (n : ℕ) : Nat.fib (n + 1) - φ * Nat.fib n = ψ ^ n := by
```

Comment:
We have a relationship between these quantities just above in `Real.coe_fib_eq`. Is there a reason you prefer to have this relationship explicitly?

Reply:
It formalises the relation given by The Art of Computer Programming: Volume 1: Fundamental Algorithms, 1.2.8. The other result also comes from Knuth but is given by David Wells Curious and Interesting Numbers, also George Bergman Number System with an Irrational Base.